### PR TITLE
Update turnserver.conf

### PR DESCRIPTION
**Pull request by hyorin**
Made some text proofreading

### CODE REVIEW

File: examples/etc/turnserver.conf [hunk 1]
```diff
@@ -663,7 +664,7 @@
 #
 #mobility
 
-# Allocate Address Family according (DEPRECATED and will be removed in favour of allocation-default-address-family)
+# Allocate Address Family according (DEPRECATED and will be removed in favor of allocation-default-address-family)
```
Comment (eakraly) [inline examples/etc/turnserver.conf:667]: Technically not an issue (English vs American)

**Comment by eakraly** [comment]
Thank you @hyorin !
Could you please do the same for `docker/coturn/turnserver.conf` file?

File: docker/coturn/turnserver.conf [hunk 2]
```diff
@@ -723,3 +766,31 @@ cli-password=CHANGE_ME
...
+# Originally, if there are more than one listener address from the same
+# address family, then by default the NAT behavior discovery feature enabled.
+# This option disables the original behavior, because the NAT behavior
+# discovery adds extra attributes to response, and this increase the
+# possibility of an amplification attack.
+#
+# Strongly encouraged to use this option to decrease gain factor in STUN
+# binding responses.
+#
+no-rfc5780
```
Comment (eakraly) [inline docker/coturn/turnserver.conf:781]: @ggarber this changes the default setting for docker containers
I do not think this is bad (better defaults - more "secure") - but it changes the default
Wanted to have another 👀 on it

Comment (ggarber) [reply docker/coturn/turnserver.conf:781]: I think it is fine.    I think the docker images are new enough that the people using rfc5780 are probably not using them.

I would land it and include it in the "Release noteS".  I'll try to take anote.

**Pull request merged**